modules/understanding-pod-error-states.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * support/investigating-pod-issues.adoc
+
+[id="understanding-pod-error-states_{context}"]
+= Understanding pod error states
+
+Pod failures return explicit error states which can be observed in the `status` field in the output of `oc get pods`. Pod error states cover image, container and container network related failures.
+
+The following table provides a list of pod error states along with their descriptions.
+
+.Pod error states
```

Comment:
Is there a way this can automatically be generated? 

@deads2k  @mfojtik

Reply:
@deads2k @mfojtik can you please confirm this? Thanks!